src/lib/features/chain_of_trust.js

```diff
@@ -65,7 +66,7 @@ class ChainOfTrust {
         });
       await fs.unlink(this.file.path);
     } catch (err) {
-      debug(err);
+      task.stream.write(fmtErrorLog(err));
```

Comment:
This writes tot the task log, right?

Reply:
Oops, you are right. Don't know why I wrote papertrail, let me change the commit message.